core/src/lib.rs

```diff
@@ -48,6 +49,11 @@ pub type PermissionTokensMap = DashMap<<Account as Identifiable>::Id, Permission
 pub type PermissionTokenDefinitionsMap =
     DashMap<<PermissionTokenDefinition as Identifiable>::Id, PermissionTokenDefinition>;
 
+/// API to work with collection of key([`DefinitionId`])-value([`HashOf<VersionedSignedTransaction>`])
+/// pairs.
```

Comment:
Can you run `cargo doc --no-deps` and check if this doc comment passes without warnings? Because I'm not sure about `HashOf<VersionedSignedTransaction>`

Reply:
This one resolved no problem, but there were some other warnings to be fixed, so I'll deal with that.